### PR TITLE
Update/readme

### PR DESCRIPTION
**Pull request by zsotroav**
I have tried updating the readme.md file to better represents the current state of the project. It is far from complete, but I want everyone to take a look at it and criticize my work. 

I ran the raw text through Grammarly, but minor grammar errors are inevitable. If you find one, report it immediately.

### CODE REVIEW

**Comment by duncte123** [comment]
Looks good to me

**Review by duncte123 — APPROVED** [review]

**Review by duncte123 — CHANGES_REQUESTED** [review]
It's missing some docker stuff:

```markdown
### docker setup

1. Install docker and docker-compose (if not installed yet)
2. Copy `docker-compose.example.yml` to `docker-compose.override.yml` (override is ignored by git)
3. Fill in the env vars in `docker-compose.override.yml`
4. Run `docker-compose up --build` to start the bot
5. Run `docker-compose down` to stop the bot (ctrl-c works but that does not clean up the container)
```

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
-# yuupy <!-- omit in toc -->
+# YuuPy <!-- omit in toc -->
+_If you are interested in joining the project as a developer, please take the time to read the whole README file and join the [development server](https://discord.gg/fPFbV8G)._
 
-_If you are interested in joining the project as a developer, please take the time to check out Kyuuto project's [docs](https://kyuuto.io/docs/)._
-
-Kyuuto bot (formerly Yuuto) is meant to be a collaboration of the [Official Camp Buddy Fan Server](https://discord.gg/campbuddy) members, completely community driven and open source. The bot's idea came from an increasing number of tech-oriented campers asking to see or contribute to _Super Hiro_ (the server's custom administrative bot).  
+YuuPy bot (formerly Yuuto or Kyuuto) is meant to be a collaboration of the [Official Camp Buddy Fan Server](https://discord.gg/campbuddy) members, completely community-driven and open source. The bot's idea came from an increasing number of tech-oriented campers asking to see or contribute to _Super Hiro_ (the server's custom administrative bot).
```
Comment (duncte123) [inline README.md:4]: Hiro got renamed :)
```suggestion
YuuPy bot (formerly Yuuto or Kyuuto) is meant to be a collaboration of the [Official Camp Buddy Fan Server](https://discord.gg/campbuddy) members, completely community-driven and open source. The bot's idea came from an increasing number of tech-oriented campers asking to see or contribute to _Hiro_ (the server's custom administrative bot).  
```

File: README.md [hunk 2]
```diff
@@ -102,59 +115,66 @@ The development server is split into different categories, which are:
...
-> **Note:** more in-depth documentation/help available on the [Kyuuto Docs](https://kyuuto.io/kb/yuuto-flow/).
+The following sections contain information about branches, and how they should be treated. If you aren't familiar with how to work with git, please familiarise yourself with its basics, and remember to ask for help from other devs should you need to. Proper git workflow and usage are crucial for the smooth operation of the project.
 
-### Master branch - Kyuuto
+### Master branch - YuuPy
 
-The _master_ branch is the most important branch and the one that contains the running code of Kyuuto (Yuuto). This branch is protected and nobody, not even the maintainers can push code to it directly. The only way code can make its way to the _master_ branch, is when _release_ gets merged with it by a maintainer.
+The _master_ branch is the most important branch and the one that contains the running code of YuuPy (Yuuto). This branch is protected, and nobody, not even the maintainers can push code to it directly. The only way code can make its way to the _master_ branch, is when _release_ gets merged with it by a maintainer.
 
 ### Release branch - BabyShark
```
Comment (duncte123) [inline README.md:138]: Changing up this weird flow a bit, the current one causes the branches to go out of sync
```suggestion
### Master branch - BabyShark
```

File: README.md [hunk 3]
```diff
@@ -102,59 +115,66 @@ The development server is split into different categories, which are:
...
 
-The _release_ branch is a branch based off _develop_ branch, and any additions to it should only contain bug fixes, quality improvements and final polishes. This branch is created once enough or noticeable features on _develop_ get finalised, and it is time to deploy them to Yuuto. The code in the _release_ branch is ran by BabyShark. Once a maintainer merges _release_ with _master_, the branch gets deleted.
+The _release_ branch is a branch based on _develop_ branch, and any additions to it should only contain bug fixes, quality improvements, and final polishes. This branch is created once enough, or noticeable features on _develop_ get finalized, and it is time to deploy them to Yuuto. The code in the _release_ branch is run by BabyShark. Once a maintainer merges _release_ with _master_, the branch gets deleted.
```
Comment (duncte123) [inline README.md:140]:
```suggestion
The _master_ branch is a branch based on _develop_ branch. This branch contains all code that would go into Yuuto before it releases so we have time to fix up some issues. All code in this branch is run by BabyShark.
```

**Review by duncte123 — CHANGES_REQUESTED** [review]

**Review by duncte123 — APPROVED** [review]

**Comment by not-archiey** [comment]
lgtm

**Review by not-archiey — APPROVED** [review]

**Pull request merged**